daemon/system-info.sh

```diff
@@ -222,7 +222,7 @@ else
         fi
 
         # If we have GNU uname, we can use that to get CPU info (probably).
-        if unmae --version 2>/dev/null | grep -qF 'GNU coreutils' ; then
+        if uname --version 2>/dev/null | grep -qF 'GNU coreutils' ; then
```

Comment:
This is scary

Reply:
Luckily this is non-critical code. The misspelling got missed in testing, and the error message it (should) generate is not likely seen by anybody because of how the script is actually used.